### DISCUSSION TITLE
LaserGRBL is not working right

### DISCUSSION THREAD

**Discussion by jacko22** [arkypita/LaserGRBL#1063] [Q&A]
[CustomButtons.gz](https://github.com/arkypita/LaserGRBL/files/4851953/CustomButtons.gz)

**Comment by gmmanonymus111** [comment]
I don't know what you are trying to do, but your "CustomButtons" seems like a LaserGRBL log file. That can't be imported.
It would be nice, if you could tell us what isn't working (what were you trying to do and what happend), some info about your machine (PC and engraver)...

**Comment by jacko22** [comment]
I'm trying to use the software laser grbl it keeps coming up as a warning grid to get in contact with you you for some reason I'm not good with the words I will try my best

**Comment by StuartB4** [comment]
Can you upload a picture of the warning.
